pkg/common/telemetry/agent/adminapi/delegatedidentity.go

```diff
@@ -0,0 +1,32 @@
+package adminapi
+
+import (
+	"github.com/spiffe/spire/pkg/common/telemetry"
+)
+
+// Call Counters (timing and success metrics)
+// Allows adding labels in-code
+
+// StartFirstUpdateLatency returns Latency metric
+// for SubscribeToX509SVIDs API fetching the first update from cache.
+func StartFirstUpdateLatency(m telemetry.Metrics) *telemetry.Latency {
+	latency := telemetry.StartLatencyMetric(m, telemetry.DelegatedIdentityAPI, telemetry.SubscribeX509SVIDs, telemetry.FirstX509SVIDUpdate)
+	return latency
```

Comment:
```suggestion
	return telemetry.StartLatencyMetric(m, telemetry.DelegatedIdentityAPI, telemetry.SubscribeX509SVIDs, telemetry.FirstX509SVIDUpdate)
```

Reply:
Looks like this is still unresolved?

Also I missed that the metric key will already include `x509_svids` from the `telemetry.SubscribeX509SVIDs` label. Maybe the last label could be something more generic like `first_update` to be more generic and shorten the metric key?

Reply:
Updated, apologies for missing it!